### DISCUSSION TITLE
Strapi logging

### DISCUSSION THREAD

**Discussion by gtamas** [strapi/strapi#6271] [Legacy Topics]
Hello all! 

Question about Strapi logging: 

is there a way to suppress log messages below a given log level? So for instance, I wanna log only warn and above. Strapi seems to log everything by default.

I tried to set STRAPI_LOG_LEVEL env var, but it has no effect.

I would like to log HTTP requests and the things I log manually (via strapi.log() ), but nothing else.

Is this possible?

**Comment by lauriejim** [comment]
Hello! We have an issue for the override of core middleware settings.
But as you can see here in the stable release code, the override works well and is fixed - [code source](https://github.com/strapi/strapi/blob/57d7d876b74efd4a5f9ba42a4062fe037cf00aae/packages/strapi/lib/core/bootstrap.js#L156)
